index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <meta charset='UTF-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <title>Tribute Page</title>
+  <link rel='stylesheet' type='text/css' href='css/style.css'>
+	<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>
+	<link href='https://fonts.googleapis.com/icon?family=Material+Icons' rel='stylesheet'>
+	<script src='https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js'></script>
+	<script src='js/script.js' type='text/javascript'></script>
+</head>
+<body>
+  <main id = 'main'>
+    <h1 id = 'title'>
+      Prof. Wangari Muta Maathai
+    </h1>
+    <blockquote cite = 'https://www.brainyquote.com/quotes/wangari_maathai_396414'>
+      <p>
+        'In a few decades, the relationship between the environment, resources and
+        conflict may seem almost as obvious as the connection we see today between
+        human rights, democracy and peace.'
+      </p>
+      <cite>- Prof. Wangari Muta Maathai</cite>
+    </blockquote>
+    <figure id='img-div'>
+      <img
+        id='image'
+        src='https://www.thoughtco.com/thmb/rDwnnZ0S34PTdzQ26QLA4bDhH8M=/3777x2501/filters:fill(auto,1)/kenyan-activist-wangari-maathai-525194310-58c341ef5f9b58af5c638e4c.jpg'
+        alt='Prof. Wangari Muta Maathai seen standing infront of a rural farm'
+        height = 740px;
+        width = 100%;
```

Comment:
Please delete the height and width, there is already a max-width and height defined in css/styles.css